1.20.5/1.20.5-pre2.md

```diff
@@ -0,0 +1,15 @@
+advancement | `Snip it!` advancement is renamed to `Shear Brilliance`
+
+data | Using ctrl+pick block no longer lists "(+NBT)" in the item tooltip
+
+assets | The animated Nether Portal texture is displayed when changing dimension to or from The Nether
+
+assets | The animated End Portal effect is displayed when changing dimension to or from The End
+
+pack breaking | The Data Pack version is now 40
```

Comment:
This should always be the first entry in the file. Also look at some of the other files to match the sentence.

Reply:
fixed.